server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -1357,8 +1357,13 @@ def uniquegroups(
         )
 
     @classmethod
-    def _build_unpivot_col(cls, *, step: 'UnpivotStep') -> Term:
-        raise NotImplementedError(f"[{cls.DIALECT}] _build_unpivot_col is not implemented")
+    def _build_unpivot_col(cls, *, step: 'UnpivotStep') -> LiteralValue:
+        unpivot_token = LiteralValue(
+            f''' UNPIVOT({cls.QUOTE_CHAR}{step.value_column_name}{cls.QUOTE_CHAR} '''
```

Comment:
Couldn't you use `pypika.Column` for that?

Reply:
Didn't find a way to avoid this %µ¨!$ issue described below. 
When using any kind of selectable in the step, `BigQuery` & `Redshift` translators generates a `FROM` clause as such 
`FROM "__step_x_yyyyy UNPIVOT(......)"` which is not valid, whereas with literal I get rid of the quotes.